docs/configure-artifact-repository.md

```diff
@@ -75,12 +75,15 @@ artifacts:
 
 ## Configuring AWS S3
 
-Create your bucket and access keys for the bucket. AWS access keys have the same
-permissions as the user they are associated with. In particular, you cannot
-create access keys with reduced scope. If you want to limit the permissions for
-an access key, you will need to create a user with just the permissions you want
-to associate with the access key. Otherwise, you can just create an access key
-using your existing user account.
+First, create your bucket:
+
+```bash
+export mybucket=bucket249
```

Comment:
```suggestion
mybucket=my-bucket-name
```

- this doesn't need exporting, since it's only used within the same shell here
- give a more example-like name so that users don't potentially think that `bucket249` is a required name (it's not)